src/cpp/shared/adventure/adv.cpp

```diff
@@ -786,6 +786,15 @@ bool GetMapCellXY(mapCell* cell, int* x, int* y) {
 	return false;
 }
 
+#include <iostream>
+
+char TileToBitmap_orig(tileset *tiles, int idx, bitmap *targ, int x, int y) {
+  if(x == -1) { // if this is removed the release build crashes
+    std::cout << idx << std::endl;
```

Comment:
Any idea why this works?

Reply:
I think that was false positive from me since it stopped working later (I think I just used debug build accidentally)